analysis/viewshed-geoprocessing/src/main/java/com/esri/samples/viewshed_geoprocessing/ViewshedGeoprocessingSample.java

```diff
@@ -103,7 +106,7 @@ public void start(Stage stage) {
       progress.setMaxWidth(30);
 
       // create the geoprocessing task with the service URL and load it
-      GeoprocessingTask geoprocessingTask = new GeoprocessingTask("https://sampleserver6.arcgisonline" +
+      geoprocessingTask = new GeoprocessingTask("https://sampleserver6.arcgisonline" +
```

Comment:
the character limit should be roughly 120 (if you haven't got it already there's a setting in IntelliJ you can use to show you the limit (File -> Settings -> Editor -> Code Style -> Hard wrap at: 120). This would be a good thing for us to doc in a Java sample contribution guide!  For this particular example,bit's way off the 120 line even when moving "new GeoprocessingTask" to a new line. So for examples like this it's fine to split for readability.

Reply:
Yep, I have this set up already. However the Java API repo and the samples repo don't necessarily have the same code style convention. Was just trying to point out the inconsistency I've been seeing between samples